openpype/settings/defaults/project_settings/global.json

```diff
@@ -3,6 +3,10 @@
         "CollectAnatomyInstanceData": {
             "follow_workfile_version": false
         },
+        "CollectAudio": {
+            "enabled": true,
```

Comment:
I would turn this plugin off by default as most of studio don't use audio and current state of plugin could do a lot of queries to mongo.